configs/autoassign/autoassign_r50_fpn_8x2_1x_coco.py

```diff
@@ -0,0 +1,82 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+model = dict(
+    type='AutoAssign',
+    pretrained='open-mmlab://detectron/resnet50_caffe',
```

Comment:
1. If we use caffe model, we should indicate that in the config.
2. Detectron caffe is deprecated, we should use detectron2/resnet50_caffe